### ISSUE TITLE
Not compatible with handlebars 4.7.0+

### ISSUE THREAD

**Issue by beaussan** [momocow/semantic-release-gitmoji#10]
Hi,

first of, I love your project, it's awesome.

I had to install it to a new project yesterday, and it didn't generate template with the emojis. 

For example, here is what was generated : 

```
# v2.0.5 (2020-02-06)

```

And, that's it. I looked into the logs and found this :

```
Handlebars: Access has been denied to resolve the property "ambulance" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
```

With a downgrade of handlebars in my lockfile, it worked again.

Can you please upgrade the dependancy level and add the corrects flags in order for this to work again on new projects ? :) 

Thanks again for your amazing tool =D

**Comment by beaussan** [comment]
If I have some time this week, I can do a pr, but I can't promise I can :)

**Comment by momocow** [comment]
Can you share your `releaseNotes` options here?

I'm also curious about what's going wrong with handlebars v4.7.0 since it is the same major version as `semantic-release-gitmoji` depends on and it should not have issues upgrading.

**Comment by beaussan** [comment]
With the default releaseNotes it dosen't work, handlebars v4.7.0 have a security fix with this flag, that should have been a major bump since it's a breaking change as the default is not the old behavior.

**Comment by beaussan** [comment]
To reproduce the bug, just update the handlebar dependancy to at least 4.7.0 and run the tests, they will fail :

```
> ava test/**/*.test.js


⠧ analyze-commits › default config + common context w/ major updates

  4 passedHandlebars: Access has been denied to resolve the property "bug" because it is not an "own property" of its parent.
⠇ analyze-commits › default config + common context w/ major updates

  4 passedHandlebars: Access has been denied to resolve the property "sparkles" because it is not an "own property" of its parent.
⠋ analyze-commits › default config + common context w/ major updates

  4 passedHandlebars: Access has been denied to resolve the property "boom" because it is not an "own property" of its parent.

  4 tests failed

  generate-notes › default config + common context w/ patch updates
    ℹ Release notes are generated.

  /home/beaussan/perso/git/semantic-release-gitmoji/test/integration/generate-notes.test.js:102

   101:   context.logger.log = t.log                                     
   102:   t.is(await generateNotes(pluginConfig, context), expectedNotes)
   103: }                                                                

  Difference:

    `# [v0.0.1](https://github.com/momocow/semantic-release-gitmoji/compare/v0.0.0...v0.0.1) (2020-02-06)␊
    ␊
  + ## 🐛 Bug Fixes␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + ␊
    `



  generate-notes › default config + common context w/ minor updates
    ℹ Release notes are generated.

  /home/beaussan/perso/git/semantic-release-gitmoji/test/integration/generate-notes.test.js:102

   101:   context.logger.log = t.log                                     
   102:   t.is(await generateNotes(pluginConfig, context), expectedNotes)
   103: }                                                                

  Difference:

    `# [v0.1.0](https://github.com/momocow/semantic-release-gitmoji/compare/v0.0.0...v0.1.0) (2020-02-06)␊
    ␊
  + ## ✨ New Features␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Add a new feature ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Add a new feature ␊
  + ␊
  + ## 🐛 Bug Fixes␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + ␊
    `



  generate-notes › default config + common context w/ major updates
    ℹ Release notes are generated.

  /home/beaussan/perso/git/semantic-release-gitmoji/test/integration/generate-notes.test.js:102

   101:   context.logger.log = t.log                                     
   102:   t.is(await generateNotes(pluginConfig, context), expectedNotes)
   103: }                                                                

  Difference:

    `# [v1.0.0](https://github.com/momocow/semantic-release-gitmoji/compare/v0.0.0...v1.0.0) (2020-02-06)␊
    ␊
  + ## ✨ New Features␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Add a new feature ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Add a new feature ␊
  + ␊
  + ## 🐛 Bug Fixes␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + ␊
  + ## 💥 Breaking Changes␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Introduce a breaking change ␊
    `



  generate-notes › default config + WIP context w/ minor updates
    ℹ Release notes are generated.

  /home/beaussan/perso/git/semantic-release-gitmoji/test/integration/generate-notes.test.js:102

   101:   context.logger.log = t.log                                     
   102:   t.is(await generateNotes(pluginConfig, context), expectedNotes)
   103: }                                                                

  Difference:

    `# [v1.0.0](https://github.com/momocow/semantic-release-gitmoji/compare/v0.0.0...v1.0.0) (2020-02-06)␊
    ␊
  + ## ✨ New Features␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Add a new feature (Issues: [`#1`](https://github.com/momocow/semantic-release-gitmoji/issues/1))␊
  +     - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Work in progress␊
  +     - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Work in progress␊
  +     - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Work in progress␊
  + ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Add a new feature (Issues: [`#2`](https://github.com/momocow/semantic-release-gitmoji/issues/2))␊
  +     - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Work in progress␊
  +     - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Work in progress␊
  +     - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Work in progress␊
  + ␊
    `

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! semantic-release-gitmoji@1.3.2 test: `ava test/**/*.test.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the semantic-release-gitmoji@1.3.2 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/beaussan/.npm/_logs/2020-02-06T11_48_17_536Z-debug.log

```

**Comment by beaussan** [comment]
I made a PR to fix this :)

**Comment by momocow** [comment]
Wow, nice!

>  handlebars v4.7.0 have a security fix with this flag

Finally I have recalled how the code is written, it's all because I use ES6 `Proxy` to resolve gitmojis by names or by unicode symbols.

https://github.com/momocow/semantic-release-gitmoji/blob/4b2b7dd931e67f8c420f77bee2551b2efabbbbbd/lib/release-notes.js#L10-L20

**Comment by momocow** [comment]
Closed this issue due to #11 merged.
